setup/pi/configure.sh

```diff
@@ -225,7 +225,7 @@ function install_archive_scripts () {
   copy_script "$archive_module"/connect-archive.sh "$install_path"
   copy_script "$archive_module"/disconnect-archive.sh "$install_path"
   copy_script "$archive_module"/archive-is-reachable.sh "$install_path"
-  if [ -n "${MUSIC_SHARE_NAME:+x}" ] && grep cifs <<< "$archive_module"
+  if [ -n "${MUSIC_SHARE_NAME:+x}" ] && [ -e "$archive_module"/copy-music.sh ]
```

Comment:
The config file documents `MUSIC_SHARE_NAME` as being the name of the CIFS share that holds the music to copy. For rsync there there is no CIFS share, so either there should be a separate variable to indicate that music should be rsync'd, or the comment in the config file should be updated.

Reply:
Well, as I for example set  `ARCHIVE_SYSTEM` to `rsync` as the comment in `teslausb_setup_variables.conf` states to do, that was the start of my confusion. I was expecting it to be the same and will adjust the comment then of course, as there is anway no way to use different archive systems for music and cam.